### ISSUE TITLE
Veture 0.19.1   Are You Kidding Me

### ISSUE THREAD

**Issue by louieaa** [vuejs/vetur#1248]
- [ ] I have searched through existing issues
- [ ] I have read through [docs](https://vuejs.github.io/vetur)
- [ ] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info
 
Property 'titleName' does not exist on type 'CombinedVueInstance<{ pageTitle: string; } & { handlePanelControl(data: any): void; handleActConfigInit(): void; } & Record<never, any> & Vue, object, object, object, Record<never, any>>'.

- Platform: 
- Vetur version: 0.19.1
- VS Code version:  1.33.1 (1.33.1)

## Problem
![image](https://user-images.githubusercontent.com/42113246/56724506-ba8ebb80-677d-11e9-821d-05ca238dcc23.png)
![image](https://user-images.githubusercontent.com/42113246/56724521-c2e6f680-677d-11e9-82ae-404fb6290432.png)
![image](https://user-images.githubusercontent.com/42113246/56724548-d09c7c00-677d-11e9-95a4-32a52a0c177e.png)

**Comment by jhgrrewq** [comment]
same problem

**Comment by louieaa** [comment]
Solution：  back version 0.18.1

**Comment by kinesias** [comment]
Temporary solution: "vetur.experimental.templateInterpolationService": false

**Comment by stefnotch** [comment]
@louieaa Would you mind posting your .vue file or a simple way to reproduce those errors?

**Comment by vinayakkulkarni** [comment]
Had to revert to 0.18.1

**Comment by lopugit** [comment]
yeah I reverted to 0.18.1 too cause it fucked up string literal colours lmfao omg

Link to the vsix from others: https://octref.gallery.vsassets.io/_apis/public/gallery/publisher/octref/extension/vetur/0.18.0/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage

**Comment by octref** [comment]
No repro case -> closing.

**Comment by lopugit** [comment]
> No repro case -> closing.

wut

open vscode, install vetur 0.19.1

make new file test.vue

type 
```
<template lang="pug">
.test(
  @click=`
    if(this){}
  `
)
</template>
```
and 
```
@click=`
    if(this){}
  `
```

and it'll look fked up

**Comment by octref** [comment]
Your issue is not even the same issue.

**Comment by v-angolenko** [comment]
Vetur version: 0.19.1

We have recently got  similar error on each $style in our template. We use CSS Modules that is provided by vue-loader. 

<img width="788" alt="Снимок экрана 2019-04-25 в 21 08 56" src="https://user-images.githubusercontent.com/49995115/56758515-80ceac80-679f-11e9-82f7-13708c0248e9.png">

**Comment by yogsky** [comment]
I have the same case, my template uses properties received through vuex's mapGetters.
all of them have the '...does not exist on type 'CombinedVueInstance' error on them 😭

**Comment by louieaa** [comment]
@stefnotch  Of course, Please update Vetur version: 0.19.1 or 0.19.0

**Comment by Farkal** [comment]
Just to say it's resolved in 0.19.2 just check VS Code have updated the extension (for my case I need to click on reload)

**Comment by stefnotch** [comment]
It's "fixed" in [0.19.2](https://github.com/vuejs/vetur/releases/tag/v0.19.2) because the experimental feature causing all of this was turned off. See https://github.com/vuejs/vetur/issues/1246

The experimental feature is still in the extension and it would be great if people would test it to make sure that everything works how it's supposed to once it gets turned on again.
